src/rgw/rgw_sal_motr.cc

```diff
@@ -2050,8 +2050,6 @@ int MotrObject::write_mobj(const DoutPrefixProvider *dpp, bufferlist&& in_buffer
 
   start = data.c_str();
   for (p = start; left > 0; left -= bs, p += bs, offset += bs) {
-    if (left < bs)
```

Comment:
This has to be removed from `MotrAtomicWriter::write()` function as well

Reply:
Done